openAPI/core-back-internal.yaml

```
@@ -111,7 +111,23 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/journeyResponse"
+                $ref: "#/components/schemas/journeyType"
+      x-amazon-apigateway-integration:
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${IPVJourneyEngineFunction.Arn}/invocations
+        passthroughBehavior: "when_no_match"
+        type: "aws_proxy"
+  /journey/cri/start/{criId}:
+    post:
+      description: Called when the frontend begins the CRI journey.
+      responses:
+        200:
+          description: "Returns the id, authorizationUrl and request for a CRI."
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/journeyType"
       x-amazon-apigateway-integration:
         httpMethod: "POST"
         uri:
```

It won't let me add a comment to the correct line here, but the line below this one needs updating to use the new AWS function name for your new lambda you've defined in the SAM template file.

```suggestion
        Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${IPVCredentialIssuerStartFunction.Arn}/invocations
```